post-result.py

```diff
@@ -22,6 +23,7 @@
 JENKINS_URL = sys.argv[2]
 BUILD_URL = sys.argv[3]
 BUILD_NAME = sys.argv[4]
+PUBLIC_JENKINS_URL = sys.argv[5]
```

Comment:
Don't you check the argc ? I meant , are those 5 parameter are all optional ?  if not, should we return error in script if no all 5 parameter given ?